config/initializers/okcomputer.rb

```diff
@@ -6,17 +6,6 @@
 
 OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
 
-# check whether resque workers are working
-OkComputer::Registry.register 'feature-resque-down', OkComputer::ResqueDownCheck.new
-
-# check for backed up resque queues: this is a low volume app, so the threshold is low
-# 2020-02-20:  Mary Ellen asked for a bigger queue so she could submit a bunch of reports at once.
-if Rails.env.production?
-  Resque.queues.each do |queue|
-    OkComputer::Registry.register "feature-#{queue}-queue-depth", OkComputer::ResqueBackedUpCheck.new(queue, 40)
-  end
-end
```

Comment:
Do we need okcomputer to do similar checks for Sidekiq or should we accommodate this in puppet using nagios or meh?

Reply:
We don't check the count in argo or dor-services-app.  As far as I know, this has never been a problem. 🤷

Reply:
Then we can defer solving it until we know we need to. 👍🏻